src/components/TeaserFront/CreditLink.js

```diff
@@ -65,12 +59,7 @@ const CreditLink = ({
 
 CreditLink.propTypes = {
   children: PropTypes.node.isRequired,
-  color: PropTypes.string,
```

Comment:
This prop is still supported right? You scared me by removing the prop type :)

Reply:
YES, still supported but I think having the string type turns `undefined` to `"undefined"`, which caused problems down the line – I'll double check that this is the case. In case it is, do you know how to fix that? (I'm not excited to do comparisons which look like `color === 'undefined' ?`)